pkg/controllers/netpol/network_policy_controller.go

```diff
@@ -560,27 +605,31 @@ func (npc *NetworkPolicyController) cleanupStaleIPSets(activePolicyIPSets map[st
 		}()
 	}
 
-	ipsets, err := utils.NewIPSet(false)
-	if err != nil {
-		return fmt.Errorf("failed to create ipsets command executor due to %s", err.Error())
-	}
-	err = ipsets.Save()
-	if err != nil {
-		klog.Fatalf("failed to initialize ipsets command executor due to %s", err.Error())
-	}
-	for _, set := range ipsets.Sets {
-		if strings.HasPrefix(set.Name, kubeSourceIPSetPrefix) ||
-			strings.HasPrefix(set.Name, kubeDestinationIPSetPrefix) {
-			if _, ok := activePolicyIPSets[set.Name]; !ok {
-				cleanupPolicyIPSets = append(cleanupPolicyIPSets, set)
-			}
+	for ipFamily := range npc.ipFamilyHandlers {
+		cleanupPolicyIPSets := make([]*utils.Set, 0)
+
+		ipsets, err := utils.NewIPSet(ipFamily)
```

Comment:
With the previous change, this must change to:

`ipsets, err := utils.NewIPSet(ipFamily.Family)`

Reply:
But I would rather use the map key directly.